website/source/docs/builders/azure.html.md

```diff
@@ -100,13 +102,130 @@ Here is a basic example for Azure.
     "capture_container_name": "images",
     "capture_name_prefix": "packer",
 
+    "os_type": "Linux",
     "image_publisher": "Canonical",
     "image_offer": "UbuntuServer",
-    "image_sku": "14.04.3-LTS",
+    "image_sku": "14.04.4-LTS",
 
     "location": "West US",
     "vm_size": "Standard_A2"
 }
 ```
 
-See the [examples/azure](https://github.com/mitchellh/packer/tree/master/examples/azure) folder in the packer project for more examples.
+## Device Login
+
+There are two ways to get started with Packer Azure. The simplest is device login, and only requires you know your
+Subscription ID. (Device login is only supported for Linux based VMs.) The second is the use of an SPN. We recommend the
+device login approach for those who are first time users, and just want to ''kick the tires.'' We recommend the SPN
+approach if you intend to automate Packer, or for deploying Windows VMs.
+
+There are three pieces of information you must provide to enable device login mode.
+
+ 1. SubscriptionID
+ 1. Resource Group - parent resource group that Packer uses to build an image.
+ 1. Storage Account - storage account where the image will be placed.
+
+> Device login mode is enabled by not setting client_id, client_secret, and tenant_id.
+
+The device login flow asks that you open a web browser, navigate to http://aka.ms/devicelogin, and input the supplied
+code. This authorizes the Packer for Azure application to act on your behalf. An OAuth token will be created, and stored
+in the user's home directory (~/.azure/packer/oauth-TenantID.json). This token is used if the token file exists, and it
+is refreshed as necessary.  The token file prevents the need to continually execute the device login flow.
+
+``` {.javascript}
+{
+    "type": "azure-arm",
+
+    "subscription_id": "44cae533-4247-4093-42cf-897ded6e7823",
+    "resource_group_name": "packerdemo",
+    "storage_account": "virtualmachines",
+
+    "capture_container_name": "images",
+    "capture_name_prefix": "packer",
+
+    "os_type": "Linux",
+    "image_publisher": "Canonical",
+    "image_offer": "UbuntuServer",
+    "image_sku": "14.04.4-LTS",
+
+    "location": "West US",
+    "vm_size": "Standard_A2"
+}
+```
+
+## Implementation
+
+This section provides details on how the Azure builder works. This section is not required to use Packer Azure, but may
```

Comment:
For these types of advanced sections we typically include a callout `~> This is an advanced topic`, like the one on [the plugins page](https://www.packer.io/docs/extend/developing-plugins.html).